lib/pbench/server/auth/auth.py

```diff
@@ -54,11 +54,7 @@ def encode_auth_token(self, time_delta: datetime.timedelta, user_id: int) -> str
 
     def get_secret_key(self):
         try:
-            return (
-                OpenIDClient.PUBLIC_KEY
-                if OpenIDClient.PUBLIC_KEY
-                else os.getenv("SECRET_KEY", "my_precious")
-            )
+            return os.getenv("SECRET_KEY", "my_precious")
         except Exception as e:
             Auth.logger.exception("Error {} getting JWT secret", e)
```

Comment:
Since `os.getenv()` doesn't raise any exceptions, I'd like to see this `try` block removed.  Is that going to happen in this PR or one of the other two?

Reply:
The entire function will refactor either in #3056 or eventually, so I am not worrying too much about it.